backend/handlers/institution_parent_handler.py

```diff
@@ -0,0 +1,101 @@
+"""Instituion Parent Handler."""
+
+from google.appengine.ext import ndb
+
+from utils import Utils
+from util import login_required, Notification, NotificationsQueueManager
+from utils import json_response
+from custom_exceptions import NotAuthorizedException
+from custom_exceptions import EntityException
+from send_email_hierarchy import RequestLinkEmailSender
+
+from models import Institution
+
+from service_messages import send_message_notification, create_system_message
+from service_entities import enqueue_task
+from util import get_subject
+
+from . import BaseHandler
+
+__all__ = ['InstitutionParentHandler']
+
+def create_system_notification(receiver_institution_key, receiver_key, notification_type):
+    message = create_system_message(receiver_institution_key)
+
+    notification = Notification(
+        message=message,
+        entity_key=receiver_institution_key.urlsafe(),
+        notification_type='ADD_ADM_PERMISSIONS',
+        receiver_key=receiver_key
+    )
+
+    notification_id = NotificationsQueueManager.create_notification_task(notification)
+    return notification_id
+
+
+class InstitutionParentHandler(BaseHandler):
+    """Institution Parent Handler."""
+
+    @json_response
+    @login_required
+    @ndb.transactional(xg=True)
+    def delete(self, user, institution_key, institution_link):
+        """
+        Handle delete parent link between institutions.
+
+        This handler remove the parent link between two institutions. 
+        If the parameter isParent is true, it means that the removal 
+        request has been made from a child institution, otherwise 
+        the request has been made by a parent institution.
+        """
+        # holds the reference of the parent intitution.
```

Comment:
now, that the method is separated, we can rename the parameters to institution_children_key and institution_parent_key and remove this doc